examples/gno.land/r/demo/groups/group.gno

```diff
@@ -0,0 +1,131 @@
+package groups
+
+import (
+	"gno.land/p/demo/avl"
+	"std"
+	"strconv"
+	"time"
+)
+
+type GroupID uint64
+type MemberID uint64
+
+func (gid GroupID) String() string {
+	return strconv.Itoa(int(gid))
```

Comment:
This may cause a problem. Since, int is 32-bit, while gid(i.e uint64) is 64-bit. 
Consider using:
```go
strconv.FormatUint(gid, 10))
```
or 
```go
fmt.Sprintf("%v", gid)
```

Reply:
Can't we recommend using machine-independent types like `int64`, `int32`..?
And about here, can we replace `strconv.Itoa` with `strconv.FormatUint`?

Reply:
The langauge itself makes use of int in many places. For loop indices by default, for example, and also methods that take indices, and more.

So we should keep int as an oft used separate component where we don't explicitly need int64.
If we need 64 bits, then we can, (e.g. block height) but in most cases this is not the case. 

Seems like we can use either in this case.

Reply:
done. kept strconv.FormatUint(gid, 10))